dist.sh

```diff
@@ -9,7 +9,7 @@ fi
 VERSION="$1"
 
 mkdir -p ~/rpmbuild/{SOURCES,SPECS}
-GZIP=-1 tar -zcf ~/rpmbuild/SOURCES/vespa-$VERSION.tar.gz --exclude target --exclude cmake-build-debug --transform "flags=r;s,^,vespa-$VERSION/," *
+GZIP=-1 tar -zcf ~/rpmbuild/SOURCES/vespa-$VERSION.tar.gz --exclude target --exclude cmake-build-debug --transform "flags=r;s,^,vespa-$VERSION/," * .*
```

Comment:
This will add . and ..

Consider using .??* instead

Reply:
Thanks. Specified the .git direcory explicitly.